core/src/sigverify.rs

```diff
@@ -128,8 +126,10 @@ impl SigVerifier for TransactionSigVerifier {
         packet_batches: Vec<PacketBatch>,
     ) -> Result<(), SigVerifyServiceError<Self::SendType>> {
         let tracer_packet_stats_to_send = std::mem::take(&mut self.tracer_packet_stats);
-        self.packet_sender
-            .send((packet_batches, Some(tracer_packet_stats_to_send)))?;
+        self.packet_sender.send(BankingPacketBatch::new((
```

Comment:
separate discussion:
@taozhu-chicago, this looks like sigverify *always* sends the tracer packet stats when sending to banking stage?
Seems odd to have an Option around them if they're always present in our main use-case. Complicates the receiving code.

Reply:
fyi, `solana_core::cluster_info_vote_listener::ClusterInfoVoteListener::check_for_leader_bank_and_send_votes()` sends `None` for the tracer packet stats. I thought that's why it's `Option<_>`-ed.

Reply:
Very right, I missed that one and was looking at the non-vote cases only.

Might still be an area to separate things in the future w/ scheduler changes as the handling for non-vote and vote txs become slightly further separated; we can use different channel types.